app/routes/_auth.login.tsx

```diff
@@ -108,14 +170,38 @@ export default function Component() {
             <button type="submit" disabled={isPending}>
               {isPending ? 'Logging in...' : 'Log In'}
             </button>
-            {actionData && actionData.message && <p className="text-red-500">{actionData.message}</p>}
+            {actionData && 'message' in actionData && <p className="text-red-500">{actionData.message}</p>}
```

Comment:
Changing `actionData.message` to `'message' in actionData` in order to avoid TS error.